pages/setup/index.vue

```diff
@@ -1,25 +1,59 @@
 <template>
     <div>
-        <div v-if="error">{{ error }}</div>
-        <div v-if="isLoading">Loading...</div>
-        <div>
+        <h1>Setting Up New Vote</h1>
+        <v-form v-model="inputValid" lazy-validation>
+            <h2>Voters List</h2>
             <div>
-                <label>Voter list:</label>
-                <v-textarea v-model="votersString" />
+                <v-textarea 
+                    v-model="votersString"
+                    solo
+                    :rules="[voterListRule]"
+                />
             </div>
+            <h2>
+                Intervals
+                <v-tooltip right>
+                    <template #activator="{ on }">
+                        <v-icon small v-on="on">mdi-information-outline</v-icon>
+                    </template>
+                    <div>1 Ethereum Block = 12 seconds</div>
+                    <div>1 Polygon Block = 5 seconds</div>
```

Comment:
We can get current network from store, then get the block time info from the network objects as in `constant`

Reply:
Done, please check [commit](https://github.com/DASC7600-2022-E-Voting/E-Voting-App/pull/23/commits/f7bfe5bd9ebf5251348becabff412214d8d0a0ad).